jormungandr-integration-tests/src/common/process_utils/output_extensions.rs

```diff
@@ -22,7 +22,11 @@ impl ProcessOutput for Output {
             let len = content.len();
             content.truncate(len - 1);
         }
-        content
+        let mut split = content.split_whitespace();
+        match split.next() {
+            Some(x) => x.to_string(),
+            None => "".to_owned(),
+        }
```

Comment:
`content.split_whitespace().next().unwrap_or_default().to_string()` maybe?
Or just `.trim()`, if you don't expect that there are multiple words, just whitespace junk.